Breaking change in `v1` module in v1.2.0

**What happened?**

The following compilation error occurs when upgrading from v1.1.2 to v1.2.0:

```
error[E0603]: struct `Timestamp` is private
    |
    |     use uuid::v1::{Context, Timestamp};
    |                             ^^^^^^^^^ private struct
    |
note: the struct `Timestamp` is defined here
   --> /code/.cargo/registry/src/github.com-1ecc6299db9ec823/uuid-1.2.0/src/v1.rs:6:22
    |
    | use crate::{Builder, Timestamp, Uuid};
    |                      ^^^^^^^^^
```

**What were you expecting?**

Because the new release is a minor release, I was expecting that the existing use of `uuid` at v1.x would be compatible with future minor releases.

**What's the context?**

- Target: `rustc 1.58.1 (db9d1b20b 2022-01-20)`
- Version: `uuid = { version = "1.0.0", default-features = false, features = ["v1"] }`

--- a/src/builder.rs
+++ b/src/builder.rs
@@ -54,7 +54,7 @@ impl Uuid {
     ///
     /// # References
     ///
-    /// * [Nil UUID in RFC4122]: https://tools.ietf.org/html/rfc4122.html#section-4.1.7
+    /// * [Nil UUID in RFC4122](https://tools.ietf.org/html/rfc4122.html#section-4.1.7)
     ///
     /// # Examples
     ///
@@ -80,7 +80,7 @@ impl Uuid {
     ///
     /// # References
     ///
-    /// * [Max UUID in Draft RFC: New UUID Formats, Version 4]: https://datatracker.ietf.org/doc/html/draft-peabody-dispatch-new-uuid-format-04#section-5.4
+    /// * [Max UUID in Draft RFC: New UUID Formats, Version 4](https://datatracker.ietf.org/doc/html/draft-peabody-dispatch-new-uuid-format-04#section-5.4)
     ///
     /// # Examples
     ///

--- a/src/v1.rs
+++ b/src/v1.rs
@@ -3,9 +3,10 @@
 //! This module is soft-deprecated. Instead of using the `Context` type re-exported here,
 //! use the one from the crate root.
 
-use crate::{Builder, Timestamp, Uuid};
+use crate::{Builder, Uuid};
 
-pub use crate::timestamp::context::Context;
+#[deprecated(note = "use types from the crate root instead")]
+pub use crate::{timestamp::context::Context, Timestamp};
 
 impl Uuid {
     /// Create a new version 1 UUID using the current system time and node ID.

